### PR TITLE
Fixed units on errors in fake TOAs

### PR DESCRIPTION
**Pull request by dlakaplan**
Previously `make_fake_toas_uniform` would ignore the units on the errors that were passed (instead, us were assumed).  I think this fixes that.

### CODE REVIEW

**Comment by abhisrkckl** [comment]
What about `make_fake_toas_fromMJDs`?

File: CHANGELOG-unreleased.md [hunk 1]
```diff
@@ -34,5 +34,6 @@ the released changes.
 - Fixed an incorrect docstring in `pbprime()` functions. 
 - Fix ICRS -> ECL conversion when parameter uncertainties are not set.
 - `get_TOAs` raises an exception upon finding mixed narrowband and wideband TOAs in a tim file. `TOAs.is_wideband` returns True only if *ALL* TOAs have the -pp_dm flag.
+- `make_fake_toas_uniform` respects units of errors
```
Comment (abhisrkckl) [inline CHANGELOG-unreleased.md:37]: Mention `make_fake_toas_fromMJDs` also.

**Comment by abhisrkckl** [comment]
This looks good to me. Shall I merge this?

**Comment by dlakaplan** [comment]
Yes, I think this is small & ready.

**Pull request merged**